server/src/main/java/org/candlepin/audit/EventBuilder.java

```diff
@@ -76,6 +76,14 @@ public EventBuilder setEventData(Eventful entity) {
                     }
                 }
             }
+            if (event.getTarget().equals(Target.POOL) && event.getType().equals(Type.CREATED)) {
+                StringBuilder eventDataBuilder = new StringBuilder()
```

Comment:
I don't want to have us get in the habit of manually building JSON strings. We can use the JsonObjectBuilder from the javax.json package, or we can build maps and pass those to Jackson (ObjectMapper) directly.